scripts/install.sh

```diff
@@ -25,3 +25,11 @@ if [[ "${TOX_ENV}" == "pypy" ]]; then
     pyenv install pypy-2.6.0
     pyenv global pypy-2.6.0
 fi
+
+if [[ "${TOX_ENV}" == "gae" && ! -d $HOME/google_appengine ]]; then
```

Comment:
Fair enough, let's use that instead. They were separate before because we use using gcloud.